libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -161,8 +160,15 @@ void AC_PrecLand::update(float rangefinder_alt_cm, bool rangefinder_alt_valid)
     struct inertial_data_frame_s inertial_data_newest;
     _ahrs.getCorrectedDeltaVelocityNED(inertial_data_newest.correctedVehicleDeltaVelocityNED, inertial_data_newest.dt);
     inertial_data_newest.Tbn = _ahrs.get_rotation_body_to_ned();
-    inertial_data_newest.inertialNavVelocity = _inav.get_velocity()*0.01f;
-    inertial_data_newest.inertialNavVelocityValid = _inav.get_filter_status().flags.horiz_vel;
+    Vector3f curr_vel;
+    _ahrs.get_velocity_NED(curr_vel);
+    curr_vel.z = -curr_vel.z;  // NED to NEU
+    inertial_data_newest.inertialNavVelocity = curr_vel;
+    nav_filter_status filt_status;
+    if (!_ahrs.get_filter_status(filt_status)) {
```

Comment:
I am not confident enough to change that much the logic. So remove the use the filter in favor of the return from get_velocity but keep the return in case it isn't good